NuGet.config

```diff
@@ -21,6 +21,7 @@
     <add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
     <add key="skiasharp" value="https://pkgs.dev.azure.com/xamarin/public/_packaging/SkiaSharp/nuget/v3/index.json" />
     <add key="wasdk-internal" value="https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json" />
+    <add key="nuget.org public" value="https://api.nuget.org/v3/index.json" />
```

Comment:
I think you have to use the `dotnet-public` feed to avoid the "dependency confusion" security issue:

https://github.com/dotnet/maui/blob/d48da938688010e0ccf248b5e6d9cae12f73c2b3/NuGet.config#L19

If you need a package added, we have to ask.

Reply:
Yea, this was most likely just added to get around some feed issues. I've rolled this file back